templates/certified/desktops.html

```diff
@@ -0,0 +1,189 @@
+{% extends "certified/base_certification.html" %}
+
+{% block title %}Certified hardware{% endblock %}
+
+{% block meta_description %}Desktops that have been certified for use with Ubuntu.{% endblock meta_description %}
```

Comment:
Are there copy docs for these pages? I feel a better description would be something like: "Ubuntu certifies thousands of desktops to guarantee they work well with Ubuntu." Gives more of a purpose of the page and content.

Reply:
FYI Dave is still chasing the PM for copy for these pages. There are rough drafts [here](https://drive.google.com/drive/folders/1a_Iz5iw7nt4VeInmHoACfCihW7ZU83bI) but they need looking at again and also need copy for the vendor specific pages

Reply:
I am actively chasing - spoke to the PM today. I've created a template for the vendor page too